docs/sources/setup-grafana/installation/debian/index.md

```diff
@@ -92,6 +92,64 @@ Complete the following steps to install Grafana using DEB or the standalone bina
 1. Depending on which system you are running, click the **Linux** or **ARM** tab on the [download page](/grafana/download).
 1. Copy and paste the code from the [download page](/grafana/download) into your command line and run.
 
+## Uninstall on Debian or Ubuntu
+
+Complete any of the following steps to uninstall Grafana.
+
+To uninstall Grafana, run any of the following commands in a terminal window:
+
+1. To uninstall Grafana OSS with systemd:
+
+   ```bash
+   sudo systemctl stop grafana-server
+   sudo apt-get remove grafana
+   ```
+
+   Optionally, you can run:
+
+   ```bash
+   sudo rm -i /etc/apt/sources.list.d/grafana.list
+   ```
```

Comment:
````suggestion
   Optional: Remove the Grafana repository

   ```bash
   sudo rm -i /etc/apt/sources.list.d/grafana.list
   ```
````

Suggest adjusting this step to use the `Optional:` formatting as discussed in the Styleguide roundtable meeting.

Also since this is a common step to all, I'd split this off as a separate step at the end.